src/argilla_llama_index/llama_index_handler.py

```diff
@@ -101,20 +56,18 @@ def __init__(
                 "Python package installed. Please install it with `pip install argilla`"
             )
 
-        # Check whether the Argilla version is compatible
         if parse(self.ARGILLA_VERSION) < parse("1.18.0"):
```

Comment:
I think we can use the internal version within the `init` to have the logic there.

Reply:
Could you elaborate a bit? This code it's already in the init.